src/ValidationErrors.cls

```diff
@@ -62,6 +65,14 @@ End Property
 Public Property Get NewEnum() As IUnknown
 Attribute NewEnum.VB_Description = "Provides custom collection support for For Each enumeration."
 Attribute NewEnum.VB_UserMemId = -4
+'QUESTION: 'VF: Can this be set via Ruberduck?
+'Attribute NewEnum.VB_Description = "Provides custom collection support for For Each enumeration."
+'Attribute NewEnum.VB_UserMemId = -4
+'I thought this were a trick I would be telling you as I think you did not mention it anywhere on your blog,  well you know them all ;-)
+'actually, this trick needs some explanations for other to understand how come this works
+'regarding collections
+'http://dailydoseofexcel.com/archives/2010/07/04/custom-collection-class/
+'Attribute NewEnum.VB_MemberFlags = "40" 'for each/next in collection

```

Comment:
FYI `VB_MemberFlags` value `40` presumably makes the member hidden in the _Object Browser_, but that flag isn't honored in VBA and in principle the `NewEnum` member would start with an underscore to hide it from the class' default interface (but underscores are an illegal identifier prefix in VBA to `NewEnum` has to remain visible).

As for how Rubberduck handles those, the [AttributeValueOutOfSync](https://rubberduckvba.com/Inspections/Details/AttributeValueOutOfSync) inspection flags hidden attributes that are out of sync with their corresponding annotation; [MissingAttribute](https://rubberduckvba.com/Inspections/Details/MissingAttribute) inspection flags annotations whose corresponding attribute isn't present, and [MissingMemberAnnotation](https://rubberduckvba.com/Inspections/Details/MissingMemberAnnotation) would flag a member that has a hidden member attribute without a corresponding annotation.

The annotation for `VB_Description` is [@Description](https://rubberduckvba.com/Annotations/Details/Description); the `VB_UserMemId = -4` value is synchronized with a [@Enumerator](https://rubberduckvba.com/Annotations/Details/EnumeratorMember) annotation (that's the magic value that enables `For Each...Next` iteration). As for `VB_UserMemId = 0`, the corresponding annotation is [@DefaultMember](https://rubberduckvba.com/Annotations/Details/DefaultMember).

See [annotations](https://rubberduckvba.com/Annotations) for the complete list, more information about each annotation, and code examples.